docs/infrastructure-mariadb.md

```diff
@@ -29,6 +19,48 @@ kubectl --namespace mariadb-system get pods -w
 
 ## Deploy the MariaDB Cluster
 
+## Pre-requsites:
+- Vault should be installed by following the instructions in [vault.md](https://github.com/rackerlabs/genestack/blob/main/docs/vault.md)
+- User has access to `osh/mariadb/` path in the Vault
+
+## Create secrets in the vault:
+
+### Login to the vault:
+```shell
```

Comment:
to keep our formatting consistent, the use of backticks should have a space after it with the type.
```suggestion
``` shell
```